src/Symfony/Bundle/FrameworkBundle/Resources/config/notifier.php

```diff
@@ -103,6 +109,10 @@
             ->args([service('texter.transports')])
             ->tag('messenger.message_handler', ['handles' => SmsMessage::class])
 
+        ->set('texter.messenger.push_handler', MessageHandler::class)
+            ->args([service('texter.transports')])
```

Comment:
Question for the Symfony maintainers (@OskarStark) - is it correct to use `texter.` here for this new push channel?  Or would we want/need a separate set of `pusher.` things instead?

Reply:
@colinodell see https://github.com/symfony/symfony/pull/39402#issuecomment-744358508
it was pusher before :)